Sources/MapboxNavigation/NavigationMapView+BuildingHighlighting.swift

```diff
@@ -11,17 +11,96 @@ extension NavigationMapView {
      
      - parameter coordinates: Coordinates which represent building locations.
      - parameter extrudesBuildings: Switch which allows to highlight buildings in either 2D or 3D. Defaults to true.
-     
-     - returns: Bool indicating if number of buildings found equals number of coordinates supplied.
+     - parameter completion: An escaping closure that operates after the buildings are highted.
+
      */
-    @discardableResult public func highlightBuildings(at coordinates: [CLLocationCoordinate2D], in3D extrudesBuildings: Bool = true) -> Bool {
-        return false
+    public func highlightBuildings(at coordinates: [CLLocationCoordinate2D],
```

Comment:
Can you cover this API with basic unit-tests?

Reply:
Thanks for the suggestion! I'll work on that.